services/src/handlers/plots.rs

```diff
@@ -56,71 +64,62 @@ pub(crate) struct GetPlot {
-///               "time": {
-///                 "start": -8334632851200000,
-///                 "end": 8210298412799999
-///               },
-///               "tilePosition": [0, 0],
-///               "globalGeoTransform": {
-///                 "originCoordinate": { "x": 0.0, "y": 0.0 },
-///                 "xPixelSize": 1.0,
-///                 "yPixelSize": -1.0
-///               },
-///               "gridArray": {
-///                 "shape": {
-///                   "shapeArray": [3, 2]
-///                 },
-///                 "data": [1, 2, 3, 4, 5, 6]
-///               }
+///               "table": [65,82,82,79,87,49,0,0,255,255,255,255,248,0,0,0,16,0,0,0,0,0,10,0,12,0,10,0,9,0,4,0,10,0,0,0,16,0,0,0,0,1,4,0,8,0,8,0,0,0,4,0,8,0,0,0,4,0,0,0,2,0,0,0,68,0,0,0,4,0,0,0,152,255,255,255,16,0,0,0,24,0,0,0,0,0,1,2,20,0,0,0,136,255,255,255,64,0,0,0,0,0,0,1,0,0,0,0,1,0,0,0,97,0,0,0,16,0,20,0,16,0,0,0,15,0,4,0,0,0,8,0,16,0,0,0,32,0,0,0,12,0,0,0,0,0,0,16,100,0,0,0,1,0,0,0,36,0,0,0,0,0,6,0,8,0,4,0,6,0,0,0,2,0,0,0,16,0,20,0,16,0,14,0,15,0,4,0,0,0,8,0,16,0,0,0,24,0,0,0,32,0,0,0,0,0,1,2,28,0,0,0,8,0,12,0,4,0,11,0,8,0,0,0,64,0,0,0,0,0,0,1,0,0,0,0,4,0,0,0,105,116,101,109,0,0,0,0,6,0,0,0,95,95,116,105,109,101,0,0,0,0,0,0,255,255,255,255,248,0,0,0,16,0,0,0,12,0,26,0,24,0,23,0,4,0,8,0,12,0,0,0,32,0,0,0,168,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3,4,0,10,0,20,0,12,0,8,0,4,0,10,0,0,0,68,0,0,0,12,0,0,0,6,0,0,0,0,0,0,0,3,0,0,0,6,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,12,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,6,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,8,0,0,0,0,0,0,0,2,0,0,0,0,0,0,0,16,0,0,0,0,0,0,0,96,0,0,0,0,0,0,0,112,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,120,0,0,0,0,0,0,0,48,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,255,0,0,0,0,0,0,0,255,255,0,0,0,0,0,0,0,40,146,17,178,99,226,255,255,23,101,16,57,43,29,0,0,40,146,17,178,99,226,255,255,23,101,16,57,43,29,0,0,40,146,17,178,99,226,255,255,23,101,16,57,43,29,0,0,40,146,17,178,99,226,255,255,23,101,16,57,43,29,0,0,40,146,17,178,99,226,255,255,23,101,16,57,43,29,0,0,40,146,17,178,99,226,255,255,23,101,16,57,43,29,0,255,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,2,0,0,0,0,0,0,0,3,0,0,0,0,0,0,0,4,0,0,0,0,0,0,0,5,0,0,0,0,0,0,0,6,0,0,0,0,0,0,0,255,255,255,255,0,0,0,0,16,0,0,0,12,0,20,0,18,0,12,0,8,0,4,0,12,0,0,0,232,0,0,0,0,1,0,0,16,0,0,0,0,0,4,0,8,0,8,0,0,0,4,0,8,0,0,0,4,0,0,0,2,0,0,0,68,0,0,0,4,0,0,0,152,255,255,255,16,0,0,0,24,0,0,0,0,0,1,2,20,0,0,0,136,255,255,255,64,0,0,0,0,0,0,1,0,0,0,0,1,0,0,0,97,0,0,0,16,0,20,0,16,0,0,0,15,0,4,0,0,0,8,0,16,0,0,0,32,0,0,0,12,0,0,0,0,0,0,16,100,0,0,0,1,0,0,0,36,0,0,0,0,0,6,0,8,0,4,0,6,0,0,0,2,0,0,0,16,0,20,0,16,0,14,0,15,0,4,0,0,0,8,0,16,0,0,0,24,0,0,0,32,0,0,0,0,0,1,2,28,0,0,0,8,0,12,0,4,0,11,0,8,0,0,0,64,0,0,0,0,0,0,1,0,0,0,0,4,0,0,0,105,116,101,109,0,0,0,0,6,0,0,0,95,95,116,105,109,101,0,0,1,0,0,0,8,1,0,0,0,0,0,0,0,1,0,0,0,0,0,0,168,0,0,0,0,0,0,0,0,0,0,0,28,1,0,0,65,82,82,79,87,49],
+///               "types": {"a": "int"}
```

Comment:
Sry, but this is too likely to change and then not work. Please use a dataset from the test data instead.

My guess it that it won't work  in the next one or two arrow versions and since this is not part of any CI, we would not notice.

Reply:
[a0893e2](https://github.com/geo-engine/geoengine/pull/667/commits/a0893e2d7a2d072dba192e0a70a35f9e4d362d79)